Example/AccessibilitySnapshot.xcodeproj/xcshareddata/xcbaselines/607FACE41AFB9204008FA782.xcbaseline/EDA548A6-E453-4CBA-9366-15D413356833.plist

```diff
@@ -11,7 +11,7 @@
 				<key>com.apple.XCTPerformanceMetric_WallClockTime</key>
 				<dict>
 					<key>baselineAverage</key>
-					<real>48.792810</real>
+					<real>17.200000</real>
```

Comment:
This brings the run time to almost (1 / scale factor), so in this case around a 65% improvement for a 3x device over the original implementation in #119.

Reply:
Increasing the `maxPermissibleMissedRegionHeight` to larger values continues to improve the run time, but you get diminishing returns.

2 => 10.259 sec
4 => 8.317 sec